packages/migrations/src/v2/migration.ts

```diff
@@ -62,34 +62,21 @@ export const runV2MigrationsAsync = async (provider: Provider, artifactsDir: str
     const secondsRequired = new BigNumber(0);
     const owner = accounts[0];
 
-    // TODO(leonid) use `AssetProxyOwner` after https://github.com/0xProject/0x-monorepo/pull/571 is merged
-    // ERC20 Multisig
-    const multiSigERC20 = await MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddressContract.deployFrom0xArtifactAsync(
-        artifacts.MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,
+    // AssetProxyOwner
+    const assetProxyOwner = await AssetProxyOwnerContract.deployFrom0xArtifactAsync(
+        artifacts.AssetProxyOwner,
         provider,
         txDefaults,
         owners,
+        [erc20proxy.address, erc721proxy.address],
         confirmationsRequired,
         secondsRequired,
-        erc20proxy.address,
     );
-    artifactsWriter.saveArtifact(multiSigERC20);
+    artifactsWriter.saveArtifact(assetProxyOwner);
     await erc20proxy.addAuthorizedAddress.sendTransactionAsync(exchange.address, { from: owner });
-    await erc20proxy.transferOwnership.sendTransactionAsync(multiSigERC20.address, { from: owner });
-
-    // ERC721 Multisig
-    const multiSigERC721 = await MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddressContract.deployFrom0xArtifactAsync(
-        artifacts.MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,
-        provider,
-        txDefaults,
-        owners,
-        confirmationsRequired,
-        secondsRequired,
-        erc721proxy.address,
-    );
-    artifactsWriter.saveArtifact(multiSigERC721);
+    await erc20proxy.transferOwnership.sendTransactionAsync(assetProxyOwner.address, { from: owner });
```

Comment:
So far, I've only been keeping track of `sendTransactionAsync` calls in the contracts package since that was the only package that interacted with Geth. If we intend to run these migrations on Geth, we should be adding `awaitTransactionSuccessAsync` for every call to `sendTransactionAsync`. If this will only ever run on Ganache, it's probably okay to leave it as is for now.

Reply:
We want to run them on Geth either. Let's slowly migrate our code (package at a time) to always await).